src/IB/IBFEMethod.cpp

```diff
@@ -991,6 +1053,7 @@ IBFEMethod::computeLagrangianFluidSource(double data_time)
     for (unsigned int part = 0; part < d_num_parts; ++part)
     {
         if (!d_lag_body_source_part[part]) continue;
+        if (!d_part_is_active[part]) continue;
```

Comment:
Merge with the previous `if`? I think it is easier to follow the logic when there is just one `continue` in a block.